content/general/limits.textile

```diff
@@ -0,0 +1,306 @@
+
+h3(#instantaneous-rates). Instantaneous rate limits
+
+Instantaneous rate limits are based on the rate of an operation per second. If the limit is exceeded then the operation will be suppressed. 
+
+Rate suppression is calculated on a rolling probabilistic basis. For example, the limit on a self-service package for publishing messages into a queue is 100 messages per second. If a queue rule is attempting to publish 200 message per second into the queue, each message will have a 50% chance of being rejected. The suppression probability is continuously updated based on the publishing rate.
+
+Instantaneous rate limits only have a hard limit. Once the hard limit rate has been exceeded then message suppression will occur. As soon as the rate drops below the hard limit threshold, the suppression probability will decrease to zero.
+
+h3(#restrictions). Account restrictions
+
+Account restrictions prevent the creation of additional resources for the limit that was hit. All other operations can continue.
+
+For example, if the peak connection hard limit is hit then no more connections can be made to Ably until some of the existing ones disconnect and the total is back below the hard limit. If an hourly hard limit is hit, such as the number of API requests, then no more API requests can be made until the following hour. 
+
+Limits that can result in an account restriction have a soft limit and a hard limit. "Quota limits":#quota use the pre-paid quota as the soft limit. A restriction doesn't come into effect until the hard limit is hit. Usage above the soft limit for self-service and business account quota limits are charged as overages. Limits unrelated to quota limits, such as token requests, can still experience an account restriction, however any usage over the soft limit is not chargeable. 
+
+Account restrictions are lifted automatically once the resource is brought below the hard limit. For connections this means disconnecting unnecessary connections and for channels, detaching from any channels not in use. Hourly-based account restrictions such as API requests are automatically lifted the following hour. 
+
+h3(#blocks). Account block
+
+An account block only occurs when the hourly or monthly message hard limits are exceeded. An account block prevents any further usage on the account for the period of the block. A block period can be for an hour or for the remainder of the current month, depending on which limit was hit.
+
+Account blocks are lifted automatically if an account package is upgraded to one with more messages. 
+
+h2(#quota). Quota limits
+
+Quotas are based on the account "package":https://ably.com/pricing/calculator that you are on. Quotas specify the number of peak channels, peak connections, messages and bandwidth that you expect to consume each month. Self-service and business accounts can configure their monthly quotas, whereas free account quotas are static and enterprise account quotas are designed to be unreachable.
```

Comment:
"Self-service and business accounts can configure their monthly quotas" -> "Monthly quotas can be configured for self-service and business accounts. Quotas for free accounts are fixed. Enterprise account quotas are set in a way that ensures that limits are not hit during normal operations for an account."

Reply:
Updated, thank you.